server/src/projects/projects.service.ts

```diff
@@ -190,6 +190,10 @@ export class ProjectsService {
         packages: projectPackages.map(pkg => ({
           ...pkg,
           invoices: joinInvoiceLabels(pkg.dcp_dcp_package_dcp_projectinvoice_package),
+
+          // NOTE: "virtual" property. Maybe not be available in other requests for pkg data!
```

Comment:
Can u explain this a bit more?

Reply:
It's generated in the API based on other values. It's not a "real" property in CRM, similar to the key names we use for entity relationships.